.github/workflows/test.yml

```diff
@@ -18,9 +19,10 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: ${{matrix.node}}
-      - uses: mymindstorm/setup-emsdk@v7
+      - uses: mymindstorm/setup-emsdk@v14
         with:
-          version: 2.0.9
```

Comment:
old emsdk looks like does not respect system nodejs versions, gts lint always return "14" for each nodejs versions

![image](https://github.com/kachick/re2-wasm/assets/1180335/046eac31-fbd3-4ee9-bee8-51ac2d87330e)